src/MICore/Debugger.cs

```diff
@@ -569,6 +571,13 @@ public Task CmdBreakInternal()
                     return CmdUnixBreak(debuggeePid, ResultClass.done);
                 }
             }
+            else if (_transport is PipeTransport)
```

Comment:
PipeTransport isn't sealed and we inherit from it for LocalTransport and ServerTransport. Should this really check the launch options to see if this is a pipe server scenario?